cmake/modules/SwiftHandleGybSources.cmake

```diff
@@ -42,6 +42,10 @@ function(handle_gyb_source_single dependency_out_var_name)
       ${GYB_SINGLE_FLAGS})
 
   set(gyb_tool "${SWIFT_SOURCE_DIR}/utils/gyb")
+
+  # GYB needs to be run via Python
+  set(gyb_tool_command "python" ${gyb_tool})
```

Comment:
Interesting suggestion, I didn't know about this. Quick question: how does this interact with multiple python versions?
I found that you can't actually compile several gyb files if you use python 3 as they rely on python 2 code (e.g. maketrans/capitalize)